backend/tests/test_tools.py

```diff
@@ -1,12 +1,16 @@
 # SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
 # SPDX-License-Identifier: Apache-2.0
 
+import pytest
+from fastapi import testclient
 
-def test_get_tools(client, project_manager):
 
+@pytest.mark.usefixtures("project_manager")
+def test_get_tools(client: testclient.TestClient, development_mode: bool):
     response = client.get("/api/v1/tools")
     out = response.json()
 
     assert response.status_code == 200
     assert "Capella" in (tool["name"] for tool in out)
-    assert "Papyrus" in (tool["name"] for tool in out)
+    if development_mode:
+        assert "Papyrus" in (tool["name"] for tool in out)
```

Comment:
I suggest the test should either run in one mode, or create two tests, one for development mode, and one for normal mode.

Reply:
I used params to create two tests now.

Reply:
Removed the Papyrus check. The test is now independent of the development mode.